locales.software/1038-hu.cfos.txt

```diff
@@ -763,7 +763,7 @@ err_os_minxp = "Az ön operációs rendszere %?.\n\n%? futtatásához Windows XP
 err_os_sup = "%p hivatalosan csak a %? vagy magasabb verziót támogatja.\n\n"
              "Lehetséges, de általunk nincs tesztelve, hogy a %p elfut-e az ön %? operációs rendszerén.\n\n"
              "Biztos szeretné folytatni a telepítést?";
-err_uninst_driver = "Az eszközmeghajtó eltávolítása sikertelen.";
+err_uninst_driver = "Az eszközmeghajtó (driver) eltávolítása sikertelen.";
```

Comment:
Hello

Yes, "eszközmeghajtó " is translation of "driver", I just wanted to stress
what it is. Many users know it by the English name "driver".

2015. december 6. 14:48 Serj Lavrin írta, <notifications@github.com>:

> In locales.software/1038-hu.cfos.txt
> <https://github.com/cFos/l10n/pull/15#discussion_r46770500>:
>
> > @@ -763,7 +763,7 @@ err_os_minxp = "Az ön operációs rendszere %?.\n\n%? futtatásához Windows XP
> >  err_os_sup = "%p hivatalosan csak a %? vagy magasabb verziót támogatja.\n\n"
> >               "Lehetséges, de általunk nincs tesztelve, hogy a %p elfut-e az ön %? operációs rendszerén.\n\n"
> >               "Biztos szeretné folytatni a telepítést?";
> > -err_uninst_driver = "Az eszközmeghajtó eltávolítása sikertelen.";
> > +err_uninst_driver = "Az eszközmeghajtó (driver) eltávolítása sikertelen.";
>
> Isn't eszközmeghajtó her already means driver?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cFos/l10n/pull/15/files#r46770500>.
>